src/app/process-editor/services/cardview-properties/called-element-item/called-element-item.component.html

```diff
@@ -0,0 +1,28 @@
+<div class="called-element-block">
+    <mat-form-field>
+        <div class="adf-property-label">{{ 'PROCESS_EDITOR.ELEMENT_PROPERTIES.ACTIVITY_NAME' | translate }}</div>
+        <mat-select (selectionChange)="changeProcess()" [(ngModel)]="processId">
+            <mat-option *ngFor="let process of processes$ | async" [value]="process.id">
+                {{ process.name }}
+            </mat-option>
+        </mat-select>
+    </mat-form-field>
+
+    <div *ngIf="processId">
+        <h5 data-automation-id="input-mapping-header">{{ 'PROCESS_EDITOR.ELEMENT_PROPERTIES.INPUT_MAPPING' | translate }}:</h5>
+        <amasdk-input-mapping-table
+            [parameters]="processVariables$ | async"
+            [processProperties]="subProcessVariables$ | async"
+            [mapping]="mapping.inputs || {}"
+            (update)="updateMapping($event, 'inputs')">
+        </amasdk-input-mapping-table>
+
+        <h5 data-automation-id="output-mapping-header">{{ 'PROCESS_EDITOR.ELEMENT_PROPERTIES.OUTPUT_MAPPING' | translate }}:</h5>
+        <amasdk-input-mapping-table
```

Comment:
This needs to be, I assume:
```
        <amasdk-output-mapping-table
            [parameters]="subProcessVariables$ | async"
            [processProperties]="processVariables$ | async"
            [mapping]="mapping.outputs || {}"
            (update)="updateMapping($event, 'outputs')">
        </amasdk-output-mapping-table>
```

Notice the other type of component, and the switch in what to inject as parameters.

Reply:
The order differs in input and output:
```
            "inputs": {
                "str1": {
                    "type": "variable",
                    "value": "str2"
                }
            },
            "outputs": {
                "str2": {
                    "type": "variable",
                    "value": "str1"
                }
            }
```

In input we specify, which parameter of the target (connector/subprocess) should be filled with which variable of the current process.
In output we specify, which variable of the current process should be filled with which variable/parameter of the target (connector/subprocess). This order is very important, if we need to keep the same constraint, as how it works for the connectors.